chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -72,7 +72,7 @@
       end
     end
 
-    arch = mnode[:kernel][:machine]
+    arch = mnode[:kernel][:machine] rescue "x86_64"
```

Comment:
Style/RescueModifier: Avoid using `rescue` in its modifier form. (https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers)